jujugui/static/gui/src/app/components/user-profile/entity/entity.js

```diff
@@ -262,25 +264,27 @@ YUI.add('user-profile-entity', function() {
         'user-profile__entity': true,
         'user-profile__list-row': true
       };
-      var isZombieModel = isModel && !entity.isAlive;
```

Comment:
Why are you removing this logic for zombie models?
We should not display the manage button if the model is not alive.
Actually, we should not display any buttons at all, nor manage neither destroy, as the model has been already destroyed.